code-challenges.test.js

```diff
@@ -11,41 +15,97 @@
 
 // --------------------INSTRUCTOR EXAMPLE: Create a function that takes in an array of numbers and returns an array with all the numbers multiplied by 3.
 
+
 // a) Create a test with expect statements for each of the variables provided.
 
 const numbersArray1 = [6, 7, 8, 9, 10]
 // Expected output: [18, 21, 24, 27, 30]
 const numbersArray2 = [24, 27, 30, 33, 36]
 // Expected output: [72, 81, 90, 99, 108]
 
+
+
+// Message for instructors:
+// I had to comment out lines 3-6 for my tests to pass. Im not sure if that was part of the asessment that we were supposed to figure out?
```

Comment:
No, those just pop up sometimes when we are using jest. It's recommended to just comment them out or delete them when they appear